pkgs/tools/audio/beets/common.nix

```diff
@@ -162,6 +162,9 @@ python3Packages.buildPythonApplication {
     "test/plugins/test_autobpm.py::TestAutoBPMPlugin::test_import"
     # AssertionError: assert 0 == 117
     "test/plugins/test_autobpm.py::TestAutoBPMPlugin::test_command"
+    # AssertionError: Image written is not /build/source/test/rsrc/abbey-similar.jpg
+    # https://github.com/beetbox/beets/pull/5650
+    "test_accept_similar_art"
```

Comment:
You know, I'd really prefer a patch here, as upstream's PR is definitely fixing this (I tested), and with a patch, on the next version bump we'd remove the patch and we'd the build will naturally continue to run this test; Whereas, with this disable the next version bumper might miss the fact this PR was merged and hence the test can be enabled again... Let's give upstream a few hours at least :pray: ? I don't see any reason they shouldn't merge that PR and hence provide us with a stable link to a patch..

Reply:
They merged it just now.